xml/art_sle_modules_quick.xml

```diff
@@ -895,8 +894,8 @@
         </row>
 -->
 
-  <sect2 xml:id="sec-modules-what-extensions">
-   <title>Extensions Available for the &sle; Product Family</title>
+ <sect1 xml:id="sec-modules-what-extensions">
+  <title>Overview of Extensions for the &sle; Product Family</title>
```

Comment:
I would shorten the titles to 'Extensions for the [...]' and 'Modules for the [...]'.

Reply:
Both occurrences fixed, thanks!